chart/templates/flower/flower-deployment.yaml

```diff
@@ -128,6 +128,8 @@ spec:
                   name: {{ template "flower_secret" . }}
                   key: basicAuth
             {{- end }}
+          envFrom:
+          {{- include "custom_airflow_environment_from" . | default "\n  []" | indent 10 }}
```

Comment:
This can't go in between stuff in `env`. Either move it above or below the whole section.

Reply:
Thanks for answering and pointing those things out. i **think** it should be better now.